### DISCUSSION TITLE
Tailwind Instead of Bootstrap Option

### DISCUSSION THREAD

**Discussion by zeliangong** [yajra/laravel-datatables#3013] [General]
Most of the Laravel Package are using Tailwind for Laravel10, would this library used tailwind css instead as an option in any future?

**Comment by yajra** [comment]
See https://datatables.net/forums/discussion/65775/integration-of-tailwind-css/p2?

I have tried to implement tailwindCss in one of my project. Doable but messy and needs a lot of work to fully support all the features.

**Reply by zeliangong** [reply]
Thank you for the confirmation and reply. Great to hear on this news
